### ISSUE TITLE
fatal: Could not open ($NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl)

### ISSUE THREAD

**Issue by lbdhr** [NCAR/ncl#57]
Hi,
 
I am a beginner of WRF. I just installed the WRFV3 and tried to run it. However, when I run the command ‘ncl plotgrids.ncl’. It always showed an error: ‘fatal: Could not open ($NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl)’. I have set the environment variable ‘export NCARG_ROOT=/usr/’. Could you please help me? Thanks very much!        The running results and self-check process is showing below:
 
uav@uav-UN65U:~/Build_WRF/WPS$ ncl plotgrids.ncl
 Copyright (C) 1995-2015 - All Rights Reserved
University Corporation for Atmospheric Research
NCAR Command Language Version 6.3.0
The use of this software is governed by a License Agreement.
See http://www.ncl.ucar.edu/ for more details.
fatal:Could not open ($NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl)
fatal:error at line 7 in file plotgrids.ncl
 
uav@uav-UN65U:~/Build_WRF/WPS$ ls $NCARG_ROOT/lib/ncarg/nclscripts/csm/gsn_code.ncl
/usr/lib/ncarg/nclscripts/csm/gsn_code.ncl
uav@uav-UN65U:~/Build_WRF/WPS$

**Comment by Asishinuzuka** [comment]
same problem with me also, please help

**Comment by rbrownrigg** [comment]
Hi,

it sounds like either NCL did not get installed directly under /usr, or
there's a permissions problem on that file.  How was NCL installed? We've
seen some issues where 3rd party packaging of NCL places things in
non-standard places, like /usr/share. Do you see an "ncarg" directory under
/usr/lib?

Rick

**Comment by Asishinuzuka** [comment]
yes, mine is also same problem and i have my NCL file in the /usr/share. Should i symlink it to /usr/lib ? i have tried symlink and it messes up my system. Is there any other way to fix this problem?

**Comment by ahmadnaveed1997** [comment]
i can see "ncarg" in my pc under lib but still facing same issue
